index.html

```diff
@@ -0,0 +1,520 @@
+                erat. Maecenas dignissim suscipit orci at molestie. Aenean
+                fringilla dolor vitae lacus lacinia eleifend. Suspendisse et
+                libero nunc. Etiam mauris tortor, pharetra [&hellip;]
+              </p>
+              <button
+                type="button"
+                class="btn btn-outline-light btn-sm text-uppercase"
+              >
+                Read More
+              </button>
+            </div>
+          </div>
+          <div class="carousel-item">
+            <img
+              src="/starter-code/img/slider-3.jpg"
+              class="d-block w-100"
+              alt="..."
+            />
+            <div class="carousel-caption d-none d-md-block w-50 mx-auto">
+              <h2 class="font-weight-light text-uppercase">
+                Fusce scelerisque nibh ac nisl luctus interdum.
+              </h2>
+              <p>
+                Nam luctus velit ultricies vestibulum eleifend. Proin imperdiet
+                varius rutrum. Curabitur placerat lacus eu ligula pellentesque
+                euismod? Aliquam volutpat augue id nibh auctor, at varius dui
+                semper? Suspendisse non pellentesque sapien, et sagittis erat.
+                Quisque et dignissim elit. Sed sit amet sapien ac quam
+                vestibulum suscipit ac id diam. Morbi accumsan est vitae tellus
+                vulputate [&hellip;]
+              </p>
+              <button
+                type="button"
+                class="btn btn-outline-light btn-sm text-uppercase"
+              >
+                Read More
+              </button>
+            </div>
+          </div>
+          <div class="carousel-item">
+            <img
+              src="/starter-code/img/slider-4.jpg"
+              class="d-block vw-100"
+              alt="..."
+            />
+            <div class="carousel-caption d-none d-md-block w-50 mx-auto">
+              <h2 class="text-uppercase">
+                Suspendisse ornare, felis feugiat suscipit pharetra
+              </h2>
+              <p>
+                Quisque sit amet sagittis justo. Suspendisse ornare, felis
+                feugiat suscipit pharetra, lorem est viverra odio; tristique
+                tristique diam tellus id tellus. In consectetur aliquet
+                tristique. Curabitur imperdiet leo in nisl consectetur mattis.
+                Quisque bibendum, mauris vitae suscipit ornare; est nibh
+                elementum mauris, eu malesuada nunc magna at justo. Proin
+                blandit id tellus non pellentesque. Mauris [&hellip;]
+              </p>
+              <button
+                type="button"
+                class="btn btn-outline-light btn-sm text-uppercase"
+              >
+                Read More
+              </button>
+            </div>
+          </div>
+        </div>
+        <a
+          class="carousel-control-prev my-auto"
+          style="
```

Comment:
Because you're using this style on two elements, I think it would have been better to create a new CSS class and use it on those elements.

Generally speaking you want to use the `style` attribute as little as possible (for debugging it's easier to have all the css defined in a css file or `<style>...</style>`).